cloudmarker/stores/emailstore.py

```diff
@@ -0,0 +1,77 @@
+from email.mime.multipart import MIMEMultipart
```

Comment:
Pylama has reported few linting issues.
```
cloudmarker/stores/emailstore.py:0:0: Incorrectly sorted imports. [isort]
cloudmarker/stores/emailstore.py:1:1: D100 Missing docstring in public module [pydocstyle]
cloudmarker/stores/emailstore.py:6:0: W0105 String statement has no effect [pylint]
cloudmarker/stores/emailstore.py:11:1: E302 expected 2 blank lines, found 1 [pycodestyle]
cloudmarker/stores/emailstore.py:11:1: D101 Missing docstring in public class [pydocstyle]
cloudmarker/stores/emailstore.py:11:0: C0111 Missing class docstring [pylint]
cloudmarker/stores/emailstore.py:13:1: D107 Missing docstring in __init__ [pydocstyle]
cloudmarker/stores/emailstore.py:13:4: R0913 Too many arguments (7/5) [pylint]
cloudmarker/stores/emailstore.py:65:9: E265 block comment should start with '# ' [pycodestyle]
cloudmarker/stores/emailstore.py:69:1: E0602 undefined name 'mailserver' [pyflakes]
cloudmarker/stores/emailstore.py:71:1: E0602 undefined name 'mailserver' [pyflakes]
cloudmarker/stores/emailstore.py:74:23: W1202 Use % formatting in logging functions and pass the % parameters as arguments [pylint]
```

You can use `make lint` command to run linter on your code. In case you want to test one specific file(e.g emailstore.py)for linting issues you can use: `pylama cloudmarker/stores/emailstore.py`.

Reply:
Done